run/storage-streaming/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "nodejs-docs-samples-http-storage-streaming",
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache-2.0",
+  "author": "Google Inc.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GoogleCloudPlatform/nodejs-docs-samples.git"
+  },
+  "engines": {
+    "node": ">=12.0.0"
+  },
+  "scripts": {
+    "test": "mocha test/*.test.js --timeout=60000 --exit"
```

Comment:
Should we add the functions frameworks commands to run?

Reply:
Done, per other thread.